docs/tutorial/spark-tutorial/t-wrapping-up.md

```diff
@@ -0,0 +1,37 @@
+## Wrapping Up
+
+### Cleanup
+
+Let's clean up the resources we created during the tutorial.
+
+First of all, let's delete the pods in the `spark` namespace.
+
+```bash
+kubectl delete pod --field-selector=status.phase==Succeeded -n spark
+kubectl delete pod --field-selector=status.phase==Failed -n spark
```

Comment:
what about deleting the namespace alltogether?

Reply:
That's a good idea.